dashboard/src/components/api/che-websocket.factory.js

```diff
@@ -22,9 +22,10 @@ export class CheWebsocket {
    * Default constructor that is using resource
    * @ngInject for Dependency injection
    */
-  constructor ($websocket, $location, proxySettings, userDashboardConfig) {
+  constructor ($websocket, $location, $interval, proxySettings, userDashboardConfig) {
```

Comment:
I think it is better to explain what `interval` means. You can rename it to pingInterval